src/components/BitcoinWallet.vue

```diff
@@ -23,13 +23,13 @@
         <CountUp
           :value="{
             endVal: walletBalance,
-            decimalPlaces: unit === 'sats' ? 0 : 5
+            decimalPlaces: unit === 'sats' ? 0 : 5,
```

Comment:
There are quite a lot of lint changes like this that are adding a lot of noise to the diff, are you able to remove them?